### ISSUE TITLE
Project Browser only showing folders and bookmarks not showing "state" items

### ISSUE THREAD

**Issue by tb-1212** [daledesilva/obsidian_project-browser#22]
Hi, 

Wondering why none of the State items are showing? I have included some screenshots

Only my folders and bookmarked items are showing. 

<img width="417" alt="SCR-20241012-jbzf" src="https://github.com/user-attachments/assets/d34ded6f-858d-4981-9d23-5ac37dec6517">
<img width="670" alt="SCR-20241012-jbxp" src="https://github.com/user-attachments/assets/77297a73-5a4f-4064-a388-adae44ec7ab4">

**Comment by daledesilva** [comment]
Looks like you might have added that state manually?

I'm pretty sure it should be "status". Try changing it to that.

Also, there should be a menu button at the top of the note that lets you change the status. But I don't see it in your screenshot.
Try disabling other plugins to see if something's hiding it.

If you're still having issues (either one), please start a new issue in GitHub and choose "bug" and fill in the template so I have the details to fix the problem.

Cheers!
Dale

**Comment by tb-1212** [comment]
oh i see the problem! I didn't know the files cannot be in a folder. For it to show the file must be in the root.

Will you consider to make it work with files that have folders?

**Comment by daledesilva** [comment]
It definitely works in folders. If you click on the folder it will show the files in there.

**Comment by tb-1212** [comment]
I mean when change the status of a file ie. Focus or Idea , the file only show up as cards when its not inside a folder. I have attached a screenshot.

When a file is placed inside a folder, even If I set their status the card does not show up in Project browser, is this something by design?

<img width="405" alt="SCR-20241012-qaij" src="https://github.com/user-attachments/assets/aa3b9484-ce10-47c7-8df9-43eb54aa3ab6">

**Comment by daledesilva** [comment]
Please watch this video and let me know if you're still having trouble.

https://www.youtube.com/live/hNz7ks87ogo?si=YzdoIZvP0yGPEgss

**Comment by technotherion** [comment]
> Looks like you might have added that state manually?
> 
> I'm pretty sure it should be "status". Try changing it to that.
> 
> Also, there should be a menu button at the top of the note that lets you change the status. But I don't see it in your screenshot. Try disabling other plugins to see if something's hiding it.
> 
> If you're still having issues (either one), please start a new issue in GitHub and choose "bug" and fill in the template so I have the details to fix the problem.
> 
> Cheers! Dale

Hey Dale, I'm on version 0.1.45. 

I used the button to add a state to a file and the property is most definitely "state". 

I experience the same behavior as @tb-1212 and think there may be confusion about what is being described above.

Here's my root folder. I have a few files set with "state: In Progress" but they do not show on this page. 
![image](https://github.com/user-attachments/assets/dae93f4f-005e-465b-be8f-fd052360e7f8)

Meanwhile, when I navigate into the "1. Projects" folder, the "1. Projects file" (a folder note that I've changed to the In Progress state) shows up. 

![image](https://github.com/user-attachments/assets/b4cec36d-5071-4f11-b3c3-5156d6517ecd)

The "Mary Jane Part 1" folder also contains a file with "state: In Progress" but only shows under the "Mary Jane Part 1" subfolder. 
![image](https://github.com/user-attachments/assets/6d030020-82c2-4376-8881-f4610f23caf9)

Now, if I understand correctly, the question is whether it's intentional that files of these states do not show on the root page of Project Browser. Each of my projects live within their own folder, so it would not be very helpful if only files from the given subdirectory were shown.

**Comment by daledesilva** [comment]
Thanks @technotherion . I believe I understand now.

Yes, this is intentional.

The project browser works like a normal file browser in that it only shows files that are in the currently selected folder. If you want to see files in sub folders, you have to go into those sub folders.

I agree, that for your use case, this makes it a little painful. But consider what would happen if we showed all files from all sub folders together... everything from your "_resources" "_templates" and "! Capture" folders would also be cluttering your view.
(By the way, you can right click on these folders and hide them if you want).

That being said, I also have the same use case. Some projects have multiple files and therefore live in a folder. These folders shouldn't live amongst the other folders, because they're what I would consider "projects".

In a future update, this  will be addressed, and you will be able to turn folders into projects so that they appear as cards with multiple pages/files.

Does that address your question as well @tb-1212 ?

**Comment by tb-1212** [comment]
Yes that's exactly what I'm looking for  @daledesilva - the ability to have project browser not just work on the "folder" level, but to have a project dashboard to collect things from different folders and shown on one page. 

May I also suggest the following 
- ability to drag and reorder files? 
- ability to open all files of the same state

I also forgot to mention, when I downloaded the plugin on "Obsidian Community plugins", some features where not accessible, ie. search and changing states on the right click menu. I had to use BRAT  to load the plugin, and those features showed up.  

And thank you @technotherion for clarifying my question!
